doc/proot/manual.txt

```diff
@@ -168,6 +168,27 @@ the sources for further details.
     group appear as if they were owned by *uid* and *gid* instead.
     Note that the ``-0`` option is the same as ``-i 0:0``.
 
+-p string, --port=string
+    Map ports to others with the syntax as *string* "port_in:port_out ...".
+
+    This option makes PRoot intercept bind and connect system calls,
+    and change the port they use. The port map is specified
+    with the syntax: ``-b *port_in*:*port_out*``. For example,
+    an application that runs a MySQL server binding to 5432 wants
+    to cohabit with other similar application, but doesn't have an
+    option to change its port. PRoot can be used here to modify
+    this port: ``proot -p 5432:5433 myapplication``. With this command,
+    the MySQL server will be bound to the port 5433.
+    This command can be repeated multiple times to map multiple ports.
+
+-a, --autoport
```

Comment:
TODO: Update the info of the commands (it's not -a but -n, and not --autoport but --netcoop).
The correct info is in src/cli/proot.h.